packages/text-sets/src/cli.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env node
```

Comment:
Add this file in the ESLint config here:

https://github.com/google/web-stories-wp/blob/6f4b205b065ed84516557d8f6d53d22cb70e33f6/.eslintrc#L381-L385

Reply:
Also one below:

https://github.com/google/web-stories-wp/blob/6f4b205b065ed84516557d8f6d53d22cb70e33f6/.eslintrc#L393-L411